### PR TITLE
Streamline config in analyzers

### PR DESCRIPTION
**Pull request by m09**
Uses a dict in both analyze and train.

### CODE REVIEW

**Comment by m09** [comment]
Related to #120 but doesn't fix it all, only the second point.

File: lookout/core/manager.py [hunk 1]
```diff
@@ -53,8 +53,10 @@ def process_review_event(self, request: ReviewEvent) -> EventResponse:
         for analyzer in self._analyzers:
             try:
                 mycfg = dict(request.configuration[analyzer.__name__])
+                self._log.info("config: %s", mycfg)
             except (KeyError, ValueError):
                 mycfg = {}
+                self._log.info("no config provided")
```
Comment (vmarkovtsev) [inline lookout/core/manager.py]: Please change to debug log level

Comment (vmarkovtsev) [reply lookout/core/manager.py]: Also message:

`"no config was provided for %s", analyzer.__name__`

File: lookout/core/manager.py [hunk 2]
```diff
@@ -53,8 +53,10 @@ def process_review_event(self, request: ReviewEvent) -> EventResponse:
         for analyzer in self._analyzers:
             try:
                 mycfg = dict(request.configuration[analyzer.__name__])
+                self._log.info("config: %s", mycfg)
```
Comment (vmarkovtsev) [inline lookout/core/manager.py]: `"%s config: %s", analyzer.__name__, mycfg`

**Pull request merged**